modules/bar/media/index.ts

```diff
@@ -50,8 +50,8 @@ const Media = () => {
                     ? `${truncatedLabel}`
                     : `${truncatedLabel.substring(0, truncatedLabel.length - 3)}...`;
         } else {
-            songIcon.value = "";
-            return "󰎇 Media 󰎇";
+            songIcon.value = getIconForPlayer(activePlayer.value?.identity || "");
+            return `${songIcon.value} Media`;
```

Comment:
```suggestion
            return `Media`;
```
Let's leave it to the icon label to handle icons. That way we have the flexibility to assigning styles to icons and configuring spacing between the icon and the label.